docs/docs.scss

```diff
@@ -281,8 +281,8 @@ body {
   .column {
     padding: $grid-gutter;
     font-family: $mono-font;
-    background-color: lighten($brand-green, 40%);
-    border: 1px solid lighten($brand-green, 30%);
+    background-color: lighten($green, 40%);
+    border: 1px solid lighten($green, 30%);
```

Comment:
Should these be variables so they can be used consistently on other elements? Something like `$bg-green-light` & `$border-green-light`